src/relay/op/dyn/tensor/transform.cc

```diff
@@ -692,6 +692,66 @@ RELAY_REGISTER_OP("dyn.expand_dims")
     .set_attr<FTVMCompute>("FTVMCompute", ExpandDimsCompute)
     .set_attr<TOpPattern>("TOpPattern", kInjective);
 
-}  // namespace dyn
-}  // namespace relay
-}  // namespace tvm
+bool DynSqueezeRel(const Array<Type>& types, int num_inputs, const Attrs& attrs,
+                const TypeReporter& reporter) {
+  ICHECK_EQ(types.size(), 3);
+  const auto* data = types[0].as<TensorTypeNode>();
+  if (data == nullptr) {
+    return false;
+  }
+  const auto* axes = types[1].as<TensorTypeNode>();
+  if (axes == nullptr) {
+    return false;
+  }
+  ICHECK_EQ(axes->shape.size(), 1);
```

Comment:
nit: add error messages to these ICHECKs

e.g. ICHECK_EQ(...) << "Error message here got " << variable << " expected 1".